base/gen_resource_account.bal

```diff
@@ -15,7 +15,7 @@
 // under the License.
 
 // AUTO-GENERATED FILE.
-// This file is auto-generated by Ballerina.
+// This file is auto-generated by WSO2 Healthcare.
```

Comment:
Shouldn't we keep this as Ballerina?? We've given Ballerina as the author in bal toml.

Reply:
Will update the codegen tool.

Reply:
All comments handled and updated the codegen tool https://github.com/wso2-enterprise/open-healthcare-integration/pull/544 as well.